bk-register/ROOTFS/opt/sage/beekeeper/register/register.py

```diff
@@ -77,6 +88,20 @@ def register():
         logger.error(e)
         return "Error: unable to register id [{}]\n".format(id), 500
 
+    # update beekeeper db
+    payload = {"node_id": id, "source": "beekeeper-register", "operation":"insert", "field_name": "registration_event", "field_value": datetime.datetime.now().replace(microsecond=0).isoformat()}
```

Comment:
I am curious.  Is there a way for the SQL table be setup to restrict the accuracy of the `datetime` to be at the seconds level?  So, you don't have to remember to set the microseconds to 0 everywhere?

Reply:
also good point, I now use TIMESTAMP(0) to disable fractional seconds in the schema